### PR TITLE
Function resources

### PR DESCRIPTION
**Pull request by qbalukom**

### CODE REVIEW

File: core-ui/src/shared/components/MemoryInput.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+// import * as jp from 'jsonpath';
+// import { FormInput, FormItem } from 'fundamental-react';
```
Comment (Wawrzyn321) [inline core-ui/src/shared/components/MemoryInput.js:3]:
```suggestion
```

File: core-ui/src/shared/components/MemoryInput.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+  const numericValue = value.match(/^\d*(\.\d*)?/)[0];
+  const unit = value.replace(numericValue, '');
+  const selectedUnit = units.includes(unit) ? unit : '';
+
+  // const setValue = val => {
+  // jp.value(container, propertyPath, val);
+  // setContainer(container);
+  // };
+
```
Comment (Wawrzyn321) [inline core-ui/src/shared/components/MemoryInput.js:26]:
```suggestion
```

**Review by Wawrzyn321 — COMMENTED** [review]
Do you think moving `RuntimeResources` presets dropdown to `CollapsiblePanel` header is worth it? We do this on `CronJobs`
image
I know we'd need to restructure the current code a bit, so... it's rather a sugestion.

**Review by Wawrzyn321 — CHANGES_REQUESTED** [review]
In Deployments create modal the `Service` section content is gone, bring it back please.

**Review by Wawrzyn321 — APPROVED** [review]

**Review by mmitoraj — APPROVED** [review]

**Pull request merged**